### ISSUE TITLE
What is the exact setting to produce the results of PETA, the code run with default setting is refer to the picture

### ISSUE THREAD

**Issue by yanghu819** [chufengt/iccv19_attribute#9]
![result](https://user-images.githubusercontent.com/44990984/80387114-d1a94180-88da-11ea-9085-27772c2fcb61.jpg)

**Comment by chufengt** [comment]
I think I've answered this question in #7.

I only used the warming-up trick (which is not mentioned in the paper, but said in  #7) when preparing the final results.

The hyper-parameters are the same as I mentioned in the paper, while I used fewer epochs (learning rate decay at 20 & 30).

**Comment by yanghu819** [comment]
thank you，I'll have a try

**Comment by syusukee** [comment]
@kickingass could you help describe how to get the PETA label files?Thanks a lot!!

**Comment by WooXinyi** [comment]
> thank you，I'll have a try

could you please share your networks settings on peta dataset? i just cannot converge my network anyway...
